src/6_branch.js

```diff
@@ -569,6 +569,9 @@ Branch.prototype['init'] = wrap(
 			self.identity = permData['identity'];
 		}
 
+		// Execute the /v1/open right away or after _web_open_delay_ms.
+		var open_delay = parseInt(utils.getParamValue('_web_open_delay_ms'), 10);
```

Comment:
Curious why you left it open to be a pattern match instead of specific one?

Reply:
I'm not sure what you mean. Do you mean the way a Regexp is used in the implementation of `utils.getParamValue`? I guess it could be made `utils.getParamValue('[?&]_web_open_delay_ms')`. This was basically modeled on the way the match ID is extracted using `utils.getParamValue('_branch_match_id')`. The idea though is that if the key is missing or otherwise can't be converted to a positive integer, it remains `undefined`, which short-circuits `utils.delay` for immediate (unqueued) execution.

Reply:
I guess what I wanted to really know was why we wanted to have the platform name as the first of the key name. I was initially thinking it'd be platform agnostic (just because it can be). So like `$web_open_delay_ms` instead of expecting the platform prepended, `$desktop_web_open_delay_ms` and in theory, it could be anything prepended, right? `blah_web_open_delay_ms` based on this function.

Not a big deal either way.

Reply:
Yeah. I guess I wanted this to be desktop-specific at the param level, hence `$desktop_web_open_delay_ms` to indicate it doesn't delay the open on redirect into an app, only on the web, and only on the desktop. But then the redirect param is implicitly web, so it could be `_open_delay_ms` instead of `_web_open_delay_ms`. I hadn't realized this function was using a regexp, so I can make it more specific. But open to alternative names, etc.

Reply:
Updated here and in the backend to `_open_delay_ms`. Also checking now for the exact key, not a regexp.